### ISSUE TITLE
Outline of elements not visible in :focus when using keyboard nav

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#270]
Will investigate and fill out more details here, just getting this down before I forget .... 

Use Chrome, and try navigating the page using the TAB key. You'll see (via the status bar) that you can navigate to each link on the page, but it is not visible on the screen what link you are on because the element is not outlined (or the outline has been made invisible). 

Suspect this has to do with something in our CSS; investigate a:focus, can use the `outline` property....

**Comment by SoLetsCode** [comment]
I tested by adding the code below to global.css. Hopefully this aids in your research.

a:focus {
  outline: 3px solid;
}

I think it will solve the issue, however when the users click the nav options (Home - Resources - Products and Services).  The outline box stays highlighted. You can see it at the end of the gif below. Not sure how to remove this effect if it's not wanted. 

![focus Kapture 2020-09-15 at 22 43 36](https://user-images.githubusercontent.com/48116904/93297132-8fa76c00-f7a5-11ea-8ef8-45575b8e1250.gif)

-Side Note-
Digital Frameworks page the links under Our Priorities 

Doesn't let users navigate to the Our Priorities links with TAB. I could fix it by adding _tabIndex={0}_ to the ScrollLink tags in digitalframework.js. Since <ScrollLink> compiles to <a> tags without an href which are not tabbable.

**Comment by mark-a-wilson** [comment]
Thanks, @SoLetsCode! 

Re: the scroll links on the Digital Framework page ... don't spend any more effort on that, as I believe the entire content of this page will be changing in the near future. 

@thegentlemanphysicist - perhaps you can add this to one of your PRs? Or would you like @SoLetsCode to submit a PR for this fix?

**Comment by mark-a-wilson** [comment]
See PR #413
